src/components/datePicker/DatesSelectionConnected.tsx

```diff
@@ -29,6 +29,8 @@ const mapStateToProps = (state: IReactVaporState, ownProps: IDatesSelectionOwnPr
   return {
     lowerLimit: item ? item.lowerLimit : new Date(),
     upperLimit: item ? item.upperLimit : new Date(),
+    inputLowerLimit: item ? item.inputLowerLimit : undefined,
+    inputUpperLimit: item ? item.inputUpperLimit : undefined,
```

Comment:
You could change this to simply `item && item.inputLowerLimit` and `item && item.inputUpperLimit`